packages/coinstac-ui/app/main/utils/boot/configure-core.js

```diff
@@ -3,7 +3,7 @@
 const app = require('ampersand-app');
 const common = require('coinstac-common');
 const CoinstacClientCore = require('coinstac-client-core');
-const merge = require('lodash/merge');
+const { merge } = require('lodash');
```

Comment:
This is mostly to make Webpack’s `externals` configuration easier. It should allow for module tree-shaking in our render process’s bundle once Webpack v2 is released.